python/equistore-operations/tests/to.py

```diff
@@ -0,0 +1,248 @@
+import numpy as np
+import pytest
+import torch
+from numpy.testing import assert_equal
+
+import equistore
+from equistore import Labels, TensorMap
+
+
+@pytest.fixture
+def block():
+    # Returns a block with nested gradients
+    values = np.arange(42, dtype=np.float64).reshape(7, 3, 2)
+    grad_values = np.arange(42, dtype=np.float64).reshape(7, 3, 2)
+    grad_grad_values = np.arange(42, dtype=np.float64).reshape(7, 3, 2)
+    block = equistore.block_from_array(values)
+    grad_block = equistore.block_from_array(grad_values)
+    grad_grad_block = equistore.block_from_array(grad_grad_values)
+    grad_block.add_gradient("grad_grad", grad_grad_block)
+    block.add_gradient("grad", grad_block)
+    return block
+
+
+@pytest.fixture
+def tmap():
```

Comment:
```suggestion
def tensor():
```

Please optimize code for readability, not ease of writing. Much more time is spent reading code than writing it: https://www.goodreads.com/quotes/835238-indeed-the-ratio-of-time-spent-reading-versus-writing-is

Reply:
Yes but that doesn't return a tensor

Reply:
```
    tmap = TensorMap(keys=Labels.arange("dummy", 2), blocks=[block1, block2])
    return tmap
```

Reply:
We use `tensor` for all fixtures returning a TensorMap: https://github.com/lab-cosmo/equistore/blob/7e8b41926ca61039f9113510daa8191270aa23c0/python/equistore-operations/tests/add.py#L15